### ISSUE TITLE
admission: disk bandwidth limiter for reads

### ISSUE THREAD

**Issue by aadityasondhi** [cockroachdb/cockroach#121780]
After we have the disk bandwidth limiter enabled for write traffic, we should design and prototype a similar limiter for read traffic. 

Prerequisite: https://github.com/cockroachdb/cockroach/issues/86857

Jira issue: CRDB-37527

**Comment by aadityasondhi** [comment]
duplicate of https://github.com/cockroachdb/cockroach/issues/107623.
